lib/read/lib/Cardano/Wallet/Read/Block/SlotNo.hs

```diff
@@ -17,9 +17,13 @@ import Cardano.Ledger.Era
     ( Era
     , EraSegWits (..)
     )
-import Cardano.Wallet.Read
+import Cardano.Wallet.Read.Block.Block
     ( Block (..)
     )
+import Cardano.Wallet.Read.Block.BlockNo
```

Comment:
probably takes instance of `Eq` or `Show` of `BlockNo` ?

Reply:
> probably takes instance of Eq or Show of BlockNo ?

Maybe. But the `BlockNo` type is not used in the module `Cardano.Wallet.Read.Block.SlotNo` at all, and the error message doesn't mention it either. If anything, it's related to instances of the `Crypto` class — but I actually suspect a compiler bug here.